public/app/plugins/datasource/loki/configuration/DebugSection.test.tsx

```diff
@@ -25,6 +27,17 @@ describe('DebugSection', () => {
     setLinkSrv(linkService);
   });
 
+  beforeEach(() => {
+    setTemplateSrv({
+      replace(target, scopedVars, format) {
+        return target ?? '';
+      },
+      getVariables() {
+        return [];
+      },
+    });
+  });
+
```

Comment:
Because of circular dependencies this test could previously run without using `setTemplateSrv`. This isn't the case any longer.